packages/opentelemetry-node/package.json

```diff
@@ -42,6 +42,7 @@
   },
   "devDependencies": {
     "@opentelemetry/context-base": "^0.6.1",
+    "@opentelemetry/plugin-http": "^0.6.1",
```

Comment:
Why was this added?

Reply:
to load the http plugin in test case

Reply:
(but test case is not complete yet as per https://github.com/open-telemetry/opentelemetry-js/pull/980#issuecomment-618671197)

Reply:
Would it make sense to add a mock plugin instead?
https://github.com/open-telemetry/opentelemetry-js/blob/23677a16f60f6dea89f449607b5549f711d6ff99/packages/opentelemetry-core/test/trace/BasePlugin.test.ts#L60

Reply:
I seem to already be using the provided mock plugin, and this is not required. Removed!
Thanks.